MatchMaker/MatchMaker/GameServers.swift

```diff
@@ -34,14 +34,77 @@ enum Dota2Servers: Servers, CustomStringConvertible {
                 
             case .SouthAmerica:
                 return NSLocalizedString("South America", comment: "South America")
-            
+                
             case .USEast:
                 return NSLocalizedString("US East", comment: "US East")
-            
+                
             case .USWest:
                 return NSLocalizedString("US West", comment: "US West")
         }
     }
+    
+    var key: String {
+        switch self {
```

Comment:
Aqui nao deveria utilizar localized strings?

Reply:
no, the keys are the ones from the localizable dictionary, it is used to store at cloudkit, so we wont to have an if, for each possible language stored at cloudkit.

Reply:
"Nao, pois a key justamente faz esse trabalho pegando o dicionário de localizable" - resposta do Lucas